package.json

```diff
@@ -28,13 +28,11 @@
   ],
   "scripts": {
     "tslint": "tslint -c tslint.json 'src/**/*.ts' -p ./ --fix",
-    "clean": "rm -rf ./lib",
     "prettier": "prettier --write 'src/**/*.ts'; prettier --write 'src/test/**/*.ts'",
-    "prepare": "npm run clean && npm run prettier && npm run tslint && tsc && cp -R src/helpers/ lib/helpers",
-    "compile": "npm run prettier && npm run tslint && tsc",
+    "prepack": "npm run tslint && tsc && cp -R src/helpers/ lib/helpers",
     "setChromeDriverVersion": "./src/scripts/setChromeDriverVersion.sh",
-    "test": "npm run compile && npm run setChromeDriverVersion && wdio src/test/wdio.conf.js --suite regression",
-    "spec": "wdio src/test/wdio.conf.js --spec",
+    "test": "DEFAULT_TIME_OUT=3000 npm run setChromeDriverVersion; npm run tslint && wdio src/test/wdio.conf.js --suite regression",
+    "spec": "DEFAULT_TIME_OUT=3000 npm run tslint && wdio src/test/wdio.conf.js --spec",
```

Comment:
may be tslint is redundant

Reply:
removing it from spec script. added it to pretest hook